skynet/modules/ttt/processor.py

```diff
@@ -96,28 +109,68 @@ def get_local_llm(**kwargs):
         model=llama_path,
         api_key='placeholder',  # use a placeholder value to bypass validation, and allow the custom base url to be used
         base_url=f'{openai_api_base_url}/v1',
-        default_headers={"X-Skynet-UUID": app_uuid},
+        default_headers={'X-Skynet-UUID': app_uuid},
         frequency_penalty=1,
         max_retries=0,
         temperature=0,
         **kwargs,
     )
 
 
+compressor = FlashrankRerank()
+
+
+async def assist(payload: DocumentPayload, customer_id: str | None = None, model: ChatOpenAI = None) -> str:
+    current_model = model or get_local_llm(max_completion_tokens=payload.max_completion_tokens)
+
+    if customer_id:
```

Comment:
Shouldn't we give an error otherwise?

Reply:
i'll remove the customer_id validation since it's redundant, we'll always have one on this branch of code, however there's still some missing implementation when there's no vector store.

Reply:
Got it. Let's give an error for now then?

Reply:
i implemented the fallback, now we just provide an empty string to `Context` when no vector store